tests/Integration/Behaviour/Features/Scenario/Order/order_odd_tax.feature

```diff
@@ -0,0 +1,297 @@
+      | total_shipping_tax_incl  | 2.42   |
+
+  Scenario: I update product quantities and prices, the price tax excluded is recomputed
+    Given order "bo_order1" does not have any invoices
+    And product "Test Product With Odd Tax" in order "bo_order1" has following details:
+      | product_quantity            | 70     |
+      | product_price               | 7.80   |
+      | original_product_price      | 7.80   |
+      | unit_price_tax_incl         | 9.438  |
+      | unit_price_tax_excl         | 7.80   |
+      | total_price_tax_excl        | 546.00 |
+      | total_price_tax_incl        | 660.66 |
+    And order "bo_order1" should have 70 products in total
+    And order "bo_order1" should have following details:
+      | total_products           | 546.00 |
+      | total_products_wt        | 660.66 |
+      | total_discounts_tax_excl | 0.0000 |
+      | total_discounts_tax_incl | 0.0000 |
+      | total_paid_tax_excl      | 548.00 |
+      | total_paid_tax_incl      | 663.08 |
+      | total_paid               | 663.08 |
+      | total_paid_real          | 0.0    |
+      | total_shipping_tax_excl  | 2.0    |
+      | total_shipping_tax_incl  | 2.42   |
+    # Edit with two values that match (with computed tax values)
+    When I edit product "Test Product With Odd Tax" to order "bo_order1" with following products details:
+      | amount         | 80    |
+      | price          | 78.00 |
+      | price_tax_incl | 94.38 |
+    Then product "Test Product With Odd Tax" in order "bo_order1" should have specific price 78.00
+    And product "Test Product With Odd Tax" in order "bo_order1" has following details:
+      | product_quantity            | 80      |
+      | product_price               | 78.00   |
+      | original_product_price      | 7.80    |
+      | unit_price_tax_incl         | 94.38   |
+      | unit_price_tax_excl         | 78.00   |
+      | total_price_tax_excl        | 6240.00 |
+      | total_price_tax_incl        | 7550.40 |
+    And order "bo_order1" should have 80 products in total
+    And order "bo_order1" should have following details:
+      | total_products           | 6240.00 |
+      | total_products_wt        | 7550.40 |
+      | total_discounts_tax_excl | 0.00000 |
+      | total_discounts_tax_incl | 0.00000 |
+      | total_paid_tax_excl      | 6242.00 |
+      | total_paid_tax_incl      | 7552.82 |
+      | total_paid               | 7552.82 |
+      | total_paid_real          | 0.0     |
+      | total_shipping_tax_excl  | 2.0     |
+      | total_shipping_tax_incl  | 2.42    |
+    # Edit with values that are not strictly equals, so price tax excluded is recomputed with additional decimals
+    When I edit product "Test Product With Odd Tax" to order "bo_order1" with following products details:
+      | amount         | 80    |
+      | price          | 78.02 |
+      | price_tax_incl | 94.40 |
+    Then product "Test Product With Odd Tax" in order "bo_order1" should have specific price 78.016528
```

Comment:
`78.02` != `78.016528` ? 🤔

Reply:
- is it because specific price amount comes from price_tax_incl ?
- should we indicate in the Behat sentence that specific price includes or not taxes ?

Reply:
`78.02` is the initial input, and `78.016528` is the recomputed value
It is managed https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Order/AbstractOrderHandler.php